java/arcs/core/host/ArcState.kt

```diff
@@ -51,7 +51,12 @@ enum class ArcState {
      * [Deleted] implies [Stopped], but furthermore, subsequent attempts to restart this
      * [Arc] will fail, and potentially any data used by the [Arc] may be reclaimed.
      */
-    Deleted
+    Deleted;
```

Comment:
semicolon

Reply:
That's required when adding members to the enum (although that will be changing as per Ray's comments above).